leapp/cli/__init__.py

```diff
@@ -7,9 +7,14 @@
 
 
 @command('')
-@command_opt('debug', is_flag=True, help='Enable debug logging', inherit=True)
+@command_opt('debug', is_flag=True, help='Enable debug mode', inherit=True)
+@command_opt('verbose', is_flag=True, help='Enable verbose logging', inherit=True)
 def cli(args):
     os.environ['LEAPP_DEBUG'] = '1' if args.debug else os.environ.get('LEAPP_DEBUG', '0')
+    if os.environ['LEAPP_DEBUG'] == '1' or args.verbose:
```

Comment:
Aren't lines 13-17 equivalent to
os.environ['LEAPP_DEBUG']  = '1' if args.verbose or args.debug or '0' ?